iroh-rpc-types/src/p2p.rs

```diff
@@ -281,6 +291,14 @@ impl Service for P2pService {
     type Res = P2pResponse;
 }
 
+impl Msg<P2pService> for WatchRequest {
+    type Response = WatchResponse;
+
+    type Update = Self;
```

Comment:
You will never get an update, but you have to put something here. A rust never type would be nice. But it is not stable yet...

https://doc.rust-lang.org/std/primitive.never.html

Reply:
@rklaehn you could expose a custom `Never` type from  quick-rpc for these cases, which is just `struct Never;` for now and can be changed to the real one later

Reply:
can this be addressed in a follow up?

Reply:
yes